geoh5py/workspace/workspace.py

```diff
@@ -609,6 +636,14 @@ def get_entity(self, name: Union[str, uuid.UUID]) -> List[Optional[Entity]]:
 
         return entity_list
 
+    @property
+    def groups(self) -> dict:
```

Comment:
**question:** why this function when there is already `all_groups()`?

Convenient to have is as a property though.
Should it just return `all_groups()`?